beps/0005-definitive-plugin-listing/README.md

```diff
@@ -166,21 +166,18 @@ interface BackstageInstance {
   };
 }
 
-interface DeploymentMetadataPluginExtensionPoint {
-  setInstances: (instances: BackstageInstance[]) => Promise<void>;
+interface DeploymentMetadataService {
```

Comment:
perhaps SystemMetadataService or such? I'm worried deployment is a bit overloaded in kubernetes where it really just means a single instance in these terms